test/cri-containerd/main.go

```diff
@@ -261,19 +261,21 @@ func convertEvent(e *types.Any) (string, interface{}, error) {
 	return id, evt, nil
 }
 
-func pullRequiredImages(t *testing.T, images []string) {
-	pullRequiredImagesWithLabels(t, images, map[string]string{
+func pullRequiredImages(t *testing.T, images []string, opts ...SandboxConfigOpt) {
+	opts = append(opts, WithSandboxLabels(map[string]string{
 		"sandbox-platform": "windows/amd64", // Not required for Windows but makes the test safer depending on defaults in the config.
-	})
+	}))
+	pullRequiredImagesWithOptions(t, images, opts...)
 }
 
-func pullRequiredLcowImages(t *testing.T, images []string) {
-	pullRequiredImagesWithLabels(t, images, map[string]string{
+func pullRequiredLcowImages(t *testing.T, images []string, opts ...SandboxConfigOpt) {
```

Comment:
nit: should be LCOW

Reply:
I think we can add LCOW and WCOW to our list of initialisms (HTTP, TLS, CPU etc.) for our stylecheck linter to check so it will catch this